kobo-uncaged/main.go

```diff
@@ -315,67 +310,9 @@ func (ku *KoboUncaged) getKoboInfo() error {
 		versFields := strings.Split(vers, ",")
 		fwStr := strings.Split(versFields[2], ".")
 		for i, f := range fwStr {
-			ku.koboInfo.fw[i], _ = strconv.Atoi(f)
+			ku.fw[i], _ = strconv.Atoi(f)
 		}
-		ku.koboInfo.model = koboDeviceID(versFields[len(versFields)-1])
```

Comment:
Do you think it might be a good idea to ensure the device actually is known here? The benefit to this is that errors parsing the file would be more visible, but the downside would be that it would be less future-proof. An alternative could be to just check the format of the device id.